gfx/portraits/portrait_modifiers/wc_infusion_appearances.txt

```diff
@@ -0,0 +1,1009 @@
+				mode = replace
+				gene = eye_color
+				x = { 0.351 0.430 }
+				y = { 0.600 0.650 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 1
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+			morph = {
+				mode = add
+				gene = gene_albino
+				template = albino_magic
+				value = 1
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = -3 }
+					has_trait = being_demon
+				}
+				NOR = {
+					has_trait = creature_orc
+					has_trait = creature_draenei
+				}
+			}
+		}
+	}
+	
+	being_demon_orc = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.351 0.430 }
+				y = { 0.600 0.650 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 1
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+			color = {
+				gene = skin_color
+				mode = replace
+				x = { 0 0.031 }
+				y = { 0.537 0.700 }
+			}
+			morph = {
+				mode = add
+				gene = gene_bs_body_shape
+				template = body_shape_triangle_full
+				value = { 0.75 1 }
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = -3 }
+					has_trait = being_demon
+				}
+				has_trait = creature_orc
+			}
+		}
+	}
+	
+	being_demon_draenei_red = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.351 0.430 }
+				y = { 0.600 0.650 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 1
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+			color = {
+				gene = skin_color
+				mode = replace
+				x = { 0 0.02 }
+				y = { 0.44 0.47 }
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = -3 }
+					has_trait = being_demon
+				}
+				has_trait = creature_draenei
+			}
+		}
+	}
+	
+	being_demon_draenei_orange = {
+		dna_modifiers = {
+			color = {
+				mode = replace
+				gene = eye_color
+				x = { 0.351 0.430 }
+				y = { 0.600 0.650 }
+			}
+			morph = {
+				mode = add
+				gene = gene_saturated_eyes
+				template = saturated_eyes
+				value = 1
+			}
+			accessory = {
+				mode = add
+				gene = special_eyes
+				template = magic_eyes
+				value = 1
+			}
+			color = {
+				gene = skin_color
+				mode = replace
+				x = { 0.047 0.067 }
+				y = { 0.44 0.5 }
+			}
+		}
+		weight = {
+			base = 0
+			modifier = {
+				add = 100
+				OR = {
+					has_infusion_level_trigger = { MAGIC = arcane_fel AMOUNT = -3 }
+					has_trait = being_demon
+				}
+				has_trait = creature_draenei
```

Comment:
Add some kind of secret gene like `demon_ethnicity` to the man'ari ethnicity.

Both draenei and man'ari have `creature_draenei` trait, in game it's localized Eredar. Thus you force man'ari to have only 2 skin colors: orange and red while there's blue skin color. Thus you need some kind of gene to see if it's man'ari or not.

If it's not man'ari, don't apply the portrait modifier.